docs/windows_developer_guide.md

```diff
@@ -96,13 +98,6 @@ npm i --global yarn
 yarn
 ```
 
-## Building with proxy
-Please set the proxy setting for windows for in environment variables when building behind proxy
-```
-set HTTP_PROXY=my.proxy.com:123
-set HTTPS_PROXY=my.proxy.com:122
-```
-
 ## GET CODE
 ```
```

Comment:
Add information thath Win11 SDK is required - on screenshot it is unselected but the opencv compilation fails